sonar-unit-tests.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Seems to choke at node-gyp with Travis. I had bit of omen about this.. Basically I might check out alternatives since now most of the things are already in place. Unless someone knows how to get the node-gyp kicking well.